consensus-core/Cargo.toml

```diff
@@ -4,7 +4,7 @@ name = "consensus-core"
 edition = "2021"
 
 [dependencies]
-alloy = { version = "0.2.1", features = ["consensus", "rpc-types", "ssz", "rlp", "k256"] }
+alloy-primitives = { version = "0.7.7", default-features = false }
```

Comment:
Note: `alloy` doesn't have `no-std` support, but `alloy-primitives` does, and this crate only needed `alloy-primitives`.